docs/apm/custom-links.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[custom-links]]
+=== Custom links
+
+Custom links allow you to easily create dynamic links based on your specific APM data.
+Links can be filtered to only appear for specific services,
+and there's no limit to the number of links you can create.
+
+[float]
+==== Create a link
+
+Create a custom link in the APM app. Navigate to **Settings** > **Customize UI**,
+and select **Create custom link**.
+
+[float]
+===== Label
+
+The name of your custom link,
+and what will be shown in the actions context menu.
+Keep it as short as possible.
+
+[float]
+===== URL
+
+The URL your link points to.
+URLs support dynamic fieldname variables, encapsulated in double curly brackets: `{{field.name}}`.
```

Comment:
Maybe we could say the variables are going to be replaced by Transaction metadata.

Reply:
Wondering if there's a good resource to link to in order to supply the user with the list of metadata variables available? We have some common ones based on ECS, but other like custom `labels`, the user will have to know what's in the Transaction document.

Reply:
Does it make sense to just point them to the metadata tab in a relevant trace sample? That will show their options for their specific data

Reply:
Sure, that would work too